### PR TITLE
New Breadcrumbs Pattern

### PR DESCRIPTION
**Pull request by beanh66**
Adding documentation for breadcrumbs pattern

### CODE REVIEW

**Comment by beanh66** [comment]
@LHinson @serenamarie125 @maryclarke @catrobson @jperry Please review when you get a chance!

File: pattern-library/navigation/breadcrumbs/design/design.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Horizontal Navigation Example
+![Horizontal Navigation Example](img/Breadcrumbs-04.png)
+
+When secondary navigation items are hidden, breadcrumbs may be used in conjunction with horizontal navigation.
+
+## Example with Optional Page Title
+![Example with Optional Page Title ](img/Breadcrumbs-05.png)
+
+1. **Current Location:** The end of the breadcrumb can also play the role of page title in an effort to conserve vertical space. If this format is used, it should be carried out for all instances of breadcrumbs in an application.
```
Comment (serenamarie125) [inline pattern-library/navigation/breadcrumbs/design/design.md]: does this read easier than the last sentence? 
"If this format is used, it should be carried out for all breadcrumb instances in an application."

File: pattern-library/navigation/breadcrumbs/design/design.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Breadcrumbs
+
+![Breadcrumbs](img/Breadcrumbs-02.png)
+
+  1. **Primary Navigation Item:** The primary navigation item should always be the first item listed in the breadcrumbs string. If the primary navigation items do not have a landing page, this item should not be listed as a link.
+  2. **Icon:**  The "fa-angle-double-right" Font Awesome icon should be used to separate levels in the breadcrumbs string.
```
Comment (serenamarie125) [inline pattern-library/navigation/breadcrumbs/design/design.md]: suggestion to change "breadcrumbs string" to "breadcrumb string"

File: pattern-library/navigation/breadcrumbs/design/design.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Breadcrumbs
+
+![Breadcrumbs](img/Breadcrumbs-02.png)
+
+  1. **Primary Navigation Item:** The primary navigation item should always be the first item listed in the breadcrumbs string. If the primary navigation items do not have a landing page, this item should not be listed as a link.
```
Comment (serenamarie125) [inline pattern-library/navigation/breadcrumbs/design/design.md]: suggestion to change "breadcrumbs string" to "breadcrumb string"

**Comment by beanh66** [comment]
@serenamarie125 Thanks for the feedback. I made those updates so the language should be consistent now :)

File: pattern-library/navigation/breadcrumbs/design/design.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Breadcrumbs
+
+![Breadcrumbs](img/Breadcrumbs-02.png)
+
```
Comment (mcarrano) [inline pattern-library/navigation/breadcrumbs/design/design.md]: It looks like in all of the examples, there is a horizontal line that separates the breadcrumb from the rest of the page content.  Is that part of this pattern?  Is this a required design element?  If so, it should be documented here.

File: pattern-library/navigation/breadcrumbs/design/design.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+## Horizontal Navigation Example
+![Horizontal Navigation Example](img/Breadcrumbs-04.png)
+
+When secondary navigation items are hidden, breadcrumbs may be used in conjunction with horizontal navigation.
+
+## Example with Optional Page Title
+![Example with Optional Page Title ](img/Breadcrumbs-05.png)
+
+1. **Current Location:** The end of the breadcrumb can also play the role of page title in an effort to conserve vertical space. If this format is used, it should be carried out for all breadcrumb instances in an application.
```
Comment (mcarrano) [inline pattern-library/navigation/breadcrumbs/design/design.md]: I like this, but we may want to elaborate more.  Looks like in this case the current page is both bold and larger?

File: pattern-library/navigation/breadcrumbs/design/design.md (hunk 4 above)
Comment (beanh66) [inline pattern-library/navigation/breadcrumbs/design/design.md]: Good point @mcarrano, I wasn't originally intending for that to be a part of this pattern. I think it depends on the rest of the page contents. For example if breadcrumbs are used in conjunction with the toolbar, the horizontal line would be there. I will update the image with callouts to note that the line is optional depending on the remaining elements in the page.

File: pattern-library/navigation/breadcrumbs/design/design.md (hunk 5 above)
Comment (beanh66) [inline pattern-library/navigation/breadcrumbs/design/design.md]: Makes sense, I will update.

**Comment by beanh66** [comment]
@mcarrano Please review the recent updates to the breadcrumbs PR :)

File: pattern-library/navigation/breadcrumbs/design/design.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Breadcrumbs
+
+![Breadcrumbs](img/Breadcrumbs-02.png)
+
+  1. **Primary Navigation Item:** The primary navigation item should always be the first item listed in the breadcrumb string. If the primary navigation items do not have a landing page, this item should not be listed as a link.
+  2. **Icon:**  The "fa-angle-double-right" Font Awesome icon should be used to separate levels in the breadcrumb string.
```
Comment (LHinson) [inline pattern-library/navigation/breadcrumbs/design/design.md]: Hey I don't like this bitmoji

**Comment by LHinson** [comment]
You're on fiyah @beanh66! 

bitmoji

Great job :)

**Comment by LHinson** [comment]
@serenamarie125 I believe your comments were addressed. Please confirm. If so, you can go ahead and merge!

**Pull request merged**